src/tetris/game.ts

```diff
@@ -4,18 +4,18 @@ import "phaser";
 import MainScene from "tetris/scene/mainScene";
 import BiasEngine from "tetris/biasEngine/biasEngine"
 import Profiler from "tetris/profiler/profiler";
-import "./styles/scss/styles.scss"
+// import "tetris/styles/scss/styles.scss"
```

Comment:
@friedow Is this right? Should we delete the line?

Reply:
This line should not be deleted. Instead, rebase the branch onto the current `developer`, which fixes the style imports and the line should work again.